src/server/pegasus_server_impl.cpp

```diff
@@ -1802,19 +1802,21 @@ ::dsn::error_code pegasus_server_impl::async_checkpoint(bool flush_memtable)
 
 // Must be thread safe.
 ::dsn::error_code pegasus_server_impl::copy_checkpoint_to_dir(const char *checkpoint_dir,
-                                                              /*output*/ int64_t *last_decree)
+                                                              /*output*/ int64_t *last_decree,
+                                                              bool flush_memtable)
```

Comment:
Usually argument is before output variables. `copy_checkpoint_to_dir(const char *checkpoint_dir, const bool flush_memtable, /*output*/ int64_t last_decree)`

Reply:
It will update all functions who call `copy_checkpoint_to_dir`, and its default value is `false`, so I just add it as the last parameter.